src/commands/start_project.rs

```diff
@@ -0,0 +1,45 @@
+async fn template_contents(project_name: &str) -> (String, String) {
+    let ftd = format!("-- import: fpm\n\n-- fpm.package: {}", project_name);
+    let index = "-- ftd.text: Hello world".to_string();
+
+    (ftd, index)
+}
+
+async fn write_file(file_name: &str, dir: &str, content: &str) -> fpm::Result<()> {
+    use tokio::io::AsyncWriteExt;
+    let file_path = format!("{}/{}", dir, file_name);
+    let mut fp = tokio::fs::File::create(file_path).await?;
+    fp.write_all(content.as_bytes()).await?;
+    Ok(())
+}
+
+pub async fn start_project(name: Option<&str>, path: Option<&str>) -> fpm::Result<()> {
+    let base_path = std::env::current_dir()?.to_str().unwrap().to_string();
+    // Not using config for base path as it requires manifest or FPM.ftd file for building and will throw error
+    // and since this command should work from anywhere within the system
+    // so we dont need to rely on config for using it
+
+    // name is a required field so it will always be some defined string (cant be None)
+    // path is an optional field and has a default value "." for current working directory (also cant be None)
+    let (name, path) = (name.unwrap(), path.unwrap());
+    let final_dir = match path {
+        "." => format!("{}/{}", base_path, name),
+        _ => format!("{}/{}/{}", base_path, path, name),
```

Comment:
Nopes, this will not work on windows. Always construct file names using `.join()` etc. Checkout `camino` crate.